src/request.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-console */
+'use strict';
+
+const http = require('http');
+
+const options = {
+  port: 8080,
+  hostname: 'localhost',
+  path: '/file/index.html',
+};
+
+http.get(options, (res) => {
```

Comment:
This logic is not needed because you should be creating a server that handles incoming requests and serves static files based on the path, not making requests.